robotpy_installer/installer.py

```diff
@@ -457,39 +530,44 @@ def ssh_from_cfg(
         pass
 
     # check to see if this is a team number
+    team = None
     try:
         team = int(hostname.strip())
     except ValueError:
-        pass
-    else:
-        # TODO: be smarter about this in the future
-        hostname = "10.%d.%d.2" % (int(team) // 100, int(team) % 100)
+        # check to see if it matches a team hostname
+        # -> allows legacy hostname configurations to benefit from
+        #    the robot finder
+        hostmod = hostname.lower().strip()
+        if not no_resolve:
+            m = re.search(r"10.(\d+).(\d+).2", hostmod)
+            if m:
+                team = int(m.group(1)) * 100 + int(m.group(2))
+            else:
+                for s in [
+                    r"roborio-(\d+)-frc.local",
+                    r"roborio-(\d+)-frc.lan",
+                    r"roborio-(\d+)-frc.frc-field.local",
```

Comment:
Wouldn't it make sense to have all this as one regex?

Also, the `.`s are unescaped.